### ISSUE TITLE
No links provided in PEGN Related Tools and Resources

### ISSUE THREAD

**Issue by tpwo** [pegn/spec#9]
Hi,

I've gone through [the webpage](https://pegn.dev/) and I noticed that there are no links provided for items in [PEGN Related Tools and Resources](https://github.com/pegn/spec/blob/2ad10e2c1bcb7230d39f1489ac983d21e4637573/doc/README.md#pegn-related-tools-and-resources):

https://github.com/pegn/spec/blob/2ad10e2c1bcb7230d39f1489ac983d21e4637573/doc/README.md#L244-L253

I think that it would be nice to have them there.

I tried to look for them, but found only [Vim](https://github.com/pegn/pegn-syntax) and [Emacs](https://github.com/pegn/pegn-mode) plugins.

**Comment by q-uint** [comment]
Hey @trivvz!

We made this list at the beginning of the project, before any of these tools/resources were made.
At this point in time only the `Vim` and `Emacs` plugins exist. I hope the others will follow soon.

- [ ] Add `Vim`/`Emacs` links to README.

**Comment by tpwo** [comment]
Thanks for the response @di-wu. I created #10 where I added these two links

**Comment by rwxrob** [comment]
Yeah, that was the plan, still is. Now that the language and spec has simmered a bit (latest addition is StateDefs) we can start looking at this more. I plan to use it for CmdBoxDoc grammar and others, but only so many weekends.
